charts/app-config/values-production.yaml

```diff
@@ -15,6 +16,64 @@ replicaCount: 3
 # Individual apps to be deployed by ArgoCD, and any values specific to those
 # apps.
 govukApplications:
+- name: asset-manager
+  chartPath: charts/asset-manager
+  helmValues:
+    workerEnabled: true
```

Comment:
This is ok as long as we're prepared to support it. Otherwise it might be better to turn off workers in prod until launch (but would need a checklist item so we don't forget!)

Reply:
Ah good shout - assume we only need the workers for publishing assets. Will turn them off for now and add a reminder.